Makefile

```diff
@@ -70,6 +70,15 @@ web-deps: web/frontend/node_modules
 web/frontend/node_modules:
 	cd web/frontend; npm install
 
+web-format: web/frontend/format
```

Comment:
`web-format` does not seem to be used anywhere.

Is it supposed to be used locally to actually apply changes and not only check as it happens on CI/CD?

Reply:
> I also think that, as a team, we will find then our style and update accordingly when and if needed.

That was actually my mindset, I just scaffolded a minimal configuration :-)

> Question: No typescript if I understand correctly, right?

Not until somebody asks for it :joy: let's keep it simple for now :grimacing:

Reply:
Also, sorry for not answering your main question: `web-format` is meant to be used only locally, what do you think about it?

Reply:
Don't be sorry :slightly_smiling_face: 
That's super, it's a nice goodie. I would just keep it.

One thing I am thinking right now is: probably we could also hook in github pre-commit and attach style checking (not fixing), so you wouldn't even be able to commit and push unstyled code.
But that's just an idea, we can introduce this if needed.